translations/handlePlainTranslations.sh

```diff
@@ -53,6 +53,31 @@ if [ $1 = "nextcloud" -a $2 = "android" ]; then
   rm -rf stable-values
 fi
 
+if [ $1 = "nextcloud" -a $2 = "talk-android" ]; then
+  mkdir stable-values
+  for version in $versions
+  do
+    git checkout $version
+    cp app/src/main/res/values/strings.xml stable-values/$version.xml
+  done
+
+  cd stable-values
+  echo '<?xml version="1.0" encoding="utf-8"?>
+  <resources>' >> combined.xml
```

Comment:
So we need to add here because it is in the original one as well: https://github.com/nextcloud/talk-android/blob/master/app/src/main/res/values/strings.xml#L23

Reply:
ae4137b